CONTRIBUTING.md

```diff
@@ -6,11 +6,77 @@ _As contributors and maintainers of this project, and in the interest of fosteri
 
 ## Getting Started
 
-We have full documentation on how to get started contributing here:
+### Repo layout
+
+```
+├── publishers # Home for all functions metadata
```

Comment:
I'd suggest the following layout to help publishing catalogs.
```
├── publishers # Home for all functions metadata
│   ├── kustomize
│   │   ├── functions
│   │   │   ├── fn-foo.yaml
│   │   │   ├── fn-bar.yaml
│   │   │   └── README.md
│   │   ├── catalogs
│   │   │   ├── v20220225.yaml
│   │   │   ├── v20220101.yaml
│   │   │   └── README.md
│   │   └── OWNERS # OWNERS of the publisher
...
```
@KnVerey WDYT?

Reply:
Hmm, yeah we didn't address where the static catalogs would go in the KEP, but it makes sense for them to be subjected to the same owners files as the functions themselves, so that layout looks good. I'd stick to a single README at the `publishers/kustomize` level though.